### PR TITLE
Added check for null value in ValueMappings and added tests

### PR DESCRIPTION
**Pull request by hugohaggmark**
Fixes #14948

### CODE REVIEW

File: packages/grafana-ui/src/components/Gauge/Gauge.test.tsx [hunk 1]
```diff
@@ -135,6 +135,45 @@ describe('Format value with value mappings', () => {
...
+    const { instance } = setup({ valueMappings });
+
+    const result = instance.getFirstFormattedValueMapping(valueMappings, value);
+
+    expect(result.text).toEqual('<NULL>');
+  });
+
+  it('should return if value is null and range to text mapping from is null', () => {
+    const valueMappings: ValueMapping[] = [
+      { id: 0, operator: '', text: '<NULL>', type: MappingType.RangeToText, from: 'null', to: '10' },
```
Comment (torkelo) [inline packages/grafana-ui/src/components/Gauge/Gauge.test.tsx]: not sure range mapping should support null, what does null to 10 mean?

**Review by torkelo — COMMENTED** [review]
This looks great but not sure we need to support null in range mappings, as it seems a bit hard to reason about a range that goes from null to some value. 

Another thing that is unrelated to this fix, could be a separate issue. Would be nice to extract the value mapping code from the Gauge visualization. Have a util function in grafana-ui/src/utils/valueMapping.ts that looks like something like getMappedValue(valueMappings: ValueMapping[], value: TimeSeriesValue)

File: packages/grafana-ui/src/components/Gauge/Gauge.test.tsx (hunk 1 above)
Comment (hugohaggmark) [reply packages/grafana-ui/src/components/Gauge/Gauge.test.tsx]: Yes, when I create a new Singlestat I get a default ValueMapping with null => N/A. If I switch to Ranged mapping instead the default ValueMapping is switched to from: null to: null => N/A so I thought we need that for ranged value mappings too. Maybe the test/code should be for from and to equal to null?

Comment (hugohaggmark) [reply packages/grafana-ui/src/components/Gauge/Gauge.test.tsx]: Will look into the getMappedValue(valueMappings: ValueMapping[], value: TimeSeriesValue) solution also :)

Comment (torkelo) [reply packages/grafana-ui/src/components/Gauge/Gauge.test.tsx]: yea, I think for range mapping to work with null, it needs to be from: null to: null, (still pretty confusing). Probably a bug in the singlestat value mapping that i defaults to that for range maps. But it does mean that singlestat thresholds probably have that out there in the wild.

Comment (hugohaggmark) [reply packages/grafana-ui/src/components/Gauge/Gauge.test.tsx]: There might be some out there in the wild yes!

**Pull request merged**